### DISCUSSION TITLE
PR title not following GroupName when only one upgrade match

### DISCUSSION THREAD

**Discussion by titilambert** [renovatebot/renovate#16843] [Request Help]
Hello,
Is there a way to force the title of a PR to use the groupName property one only one upgrade match the package rule ?

Thanks

**Comment by rarkins** [comment]
No simple option for this, it could potentially be possible by manipulating `prTitle` directly

**Reply by titilambert** [reply]
Thanks ! It worked perfectly with `commitMessageTopic` since `prTitle` is deprecated
